playground/components/header.tsx

```diff
@@ -108,6 +109,18 @@ export const Header: ParentComponent<{
           <Icon class="h-6" path={copy() ? link : share} />
           <span class="text-xs md:sr-only">{copy() ? 'Copied to clipboard' : 'Share'}</span>
         </button>
+        <a
+          href="https://github.com/solidjs/solid-playground"
+          target="_blank"
+          class="flex flex-row items-center space-x-2 rounded px-2 py-2 opacity-80 hover:opacity-100 md:px-1"
+          classList={{
+            'rounded-none	active:bg-gray-300 hover:bg-gray-300 dark:hover:text-black': showMenu(),
```

Comment:
There's an odd tab hiding here that should be replaced with a space (running `pnpm run format` might fix that, but it may have to be done manually)